common/changes/@microsoft/rush/main_2022-09-20-13-40.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Add commit flag that commits the change files.",
+      "type": "none"
```

Comment:
Should this be `minor`? Since you effectively extended the API

Reply:
Because Rush is lockstep versioned, bump-types aren't passed in here.